test/integration/targets/connection_ssh/test_ssh_askpass.yml

```diff
@@ -3,22 +3,20 @@
     - import_role:
         role: setup_test_user
 
-    - find:
-        paths: /etc/ssh
-        recurse: true
-        contains: 'PasswordAuthentication'
-      register: sshd_confs
-
-    - lineinfile:
-        path: '{{ item }}'
-        regexp: '^PasswordAuthentication'
-        line: PasswordAuthentication yes
-      loop: '{{ sshd_confs.files|default([{"path": "/etc/ssh/sshd_config"}], true)|map(attribute="path") }}'
-
-    - when: ansible_facts.system == 'Darwin'
+    # macos currently allows password auth, and macos/15.3 prevents restarting sshd
```

Comment:
Does it? The VM is supposed to grant full disk access to the administrator. Could you add more details here?

Reply:
```
$ ansible-test shell -v --remote macos/15.3
[...]
root@administrators-Mac-3:~/ansible# launchctl stop com.openssh.sshd
Not privileged to stop service.
```

This works on the macos/14.3 remote, and works on my local `15.0.1` that I continuously forget to update :)

Reply:
:thinking: I think local would be different since it wouldn't require full-disk access over SSH, though. When I was updating macOS hosts on MacStadium to newer versions, I was also hitting privilege problems post-reboot that required their stuff to physically walk to the machines and use the local keyboard-level access to enable the respective privileges.